pandas/io/formats/format.py

```diff
@@ -53,35 +53,38 @@
     Parameters
     ----------
     buf : StringIO-like, optional
-        buffer to write to
-    columns : sequence, optional
-        the subset of columns to write; default None writes all columns
+        Buffer to write to.
+    columns : sequence, optional, default None
+        The subset of columns to write. The default None writes all columns.
     col_space : int, optional
-        the minimum width of each column
+        The minimum width of each column.
     header : bool, optional
-        %(header)s
-    index : bool, optional
-        whether to print index (row) labels, default True
-    na_rep : string, optional
-        string representation of NAN to use, default 'NaN'
-    formatters : list or dict of one-parameter functions, optional
-        formatter functions to apply to columns' elements by position or name,
-        default None. The result of each function must be a unicode string.
+        %(header)s.
```

Comment:
Substituting the header here won't break lines at 79 characters. Not sure this is worth fixing though.